README.md

```diff
@@ -103,57 +103,126 @@ FLP is a framework. The main FLP object is the **pipeline**. FLP **pipeline** ca
 - **ingest** - obtain flows from some source, one entry per line
 - **decode** - parse input lines into a known format, e.g., dictionary (map) of AWS or goflow data
 - **transform** - convert entries into a standard format; can include multiple transform stages
 - **write** - provide the means to write the data to some target, e.g. loki, standard output, etc
 - **extract** - derive a set of metrics from the imported flows
 - **encode** - make the data available in appropriate format (e.g. prometheus)
 
-The **encode** and **write** stages may be combined in some cases (as in prometheus), in which case **write** is set to **none**
+The first stage in a pipeline must be an **ingest** stage, followed by a **decode** stage.
+Each stage (other than an **ingest** stage) specifies the stage it follows.
+Multiple stages may follow from a particular stage, thus allowing the same data to be fed into multiple parallel pipelines.
+Multiple **transform** stages may be performed and the results may be output to different targets.
 
+A full configuration file with the data fed into two different transforms might look like the following.
+
+```yaml
+pipeline:
+  - name: ingest1
+  - name: decode1
+    follows: ingest1
+  - name: generic1
+    follows: decode1
+  - name: write1
+    follows: generic1
+  - name: generic2
+    follows: decode1
+  - name: write2
+    follows: generic2
+parameters:
+  - name: decode1
+    decode:
+      type: json
+  - name: generic1
+    transform:
+      type: generic
+      generic:
+        rules:
+          - input: Bytes
+            output: v1_bytes
+          - input: DstAddr
+            output: v1_dstAddr
+          - input: Packets
+            output: v1_packets
+          - input: SrcPort
+            output: v1_srcPort
+  - name: write1
+    write:
+      type: stdout
+  - name: generic2
+    transform:
+      type: generic
+      generic:
+        rules:
+          - input: Bytes
+            output: v2_bytes
+          - input: DstAddr
+            output: v2_dstAddr
+          - input: Packets
+            output: v2_packets
+          - input: SrcPort
+            output: v2_srcPort
+  - name: ingest1
+    ingest:
+      type: file_loop
+      file:
+        filename: playground/goflow2_input.txt
+  - name: write2
+    write:
+      type: stdout
+```
 It is expected that the **ingest** module will receive flows every so often, and this ingestion event will then trigger the rest of the pipeline. So, it is the responsibility of the **ingest** module to provide the timing of when (and how often) the pipeline will run.
 
 # Configuration
 
 It is possible to configure flowlogs-pipeline using command-line-parameters, configuration file, environment variables, or any combination of those options.
 
+
 For example:
 1. Using command line parameters:
-```./flowlogs-pipeline --pipeline.ingest.type file --pipeline.ingest.file.filename hack/examples/ocp-ipfix-flowlogs.json```
+```./flowlogs-pipeline --log-level info ```
```

Comment:
Will that work ??? don't we need to define the input somehow ... this might be a longs JSON now, but still we can do that from the command line

Reply:
Added --config <configFile>. The command-line parameters over-ride the config file parameters.